indexer/services/comlink/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "outDir": "build",
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
+    "skipLibCheck": true
```

Comment:
what does this do?

Reply:
Turns out `@keplr-wallet/cosmos` will throw ts errors, unless `"skipLibCheck": true`.

Reply:
During compile, `@keplr-wallet/cosmos` throws many type errors. The simplest resolution is to `"skipLibCheck": true`, which will skip type checking for dependency code that we do not reference.
```
https://www.typescriptlang.org/tsconfig/#skipLibCheck
Skip type checking of declaration files.

This can save time during compilation at the expense of type-system accuracy. For example, two libraries could define two copies of the same type in an inconsistent way. Rather than doing a full check of all d.ts files, TypeScript will type check the code you specifically refer to in your app’s source code.
```